_delphi_utils_python/delphi_utils/validator/datafetcher.py

```diff
@@ -3,19 +3,20 @@
 
 import re
 import threading
+import os
 from os import listdir
 from os.path import isfile, join
 import warnings
 import requests
 import pandas as pd
 import numpy as np
-
 import covidcast
 from .errors import APIDataFetchError, ValidationFailure
 
 FILENAME_REGEX = re.compile(
     r'^(?P<date>\d{8})_(?P<geo_type>\w+?)_(?P<signal>\w+)\.csv$')
 
+API_KEY = ("api_key",os.environ.get("COVIDCAST_INDICATORS_KEY"))
```

Comment:
youll wanna use the "epidata" username for this type of auth
```suggestion
API_KEY = ("epidata", os.environ.get("COVIDCAST_INDICATORS_KEY"))
```
(see: [delphi-epidata:src/server/_security.py](https://github.com/cmu-delphi/delphi-epidata/blob/14fb1c189b49335f5e529eab96956f3393e41e6c/src/server/_security.py#L34) )

also, do you want to warn or error if this is not present in the environment?  similarly, youll probably want to change the `auth` arg  in the calls to `requests.get()` below if its not....